relay-dynamic-config/src/global.rs

```diff
@@ -12,11 +12,11 @@ use crate::TaggingRule;
 pub struct GlobalConfig {
     /// Project configuration for measurements.
     #[serde(skip_serializing_if = "Option::is_none")]
-    measurements: Option<MeasurementsConfig>,
+    pub measurements: Option<MeasurementsConfig>,
     /// Project configuration for rules for applying metrics tags depending on
     /// the event's content.
-    #[serde(skip_serializing_if = "Option::is_none")]
-    metric_conditional_tagging: Option<Vec<TaggingRule>>,
+    #[serde(skip_serializing_if = "Vec::is_empty")]
```

Comment:
I think vectors dont need to be optional wrt version compatibility right? since it'll just use an empty one if its missing

Reply:
The `#[serde(default)]` on the `struct` should be enough:

> When deserializing, any missing fields should be filled in from the struct's implementation of Default. Only allowed on structs.

https://serde.rs/container-attrs.html